.env.example

```diff
@@ -0,0 +1,4 @@
+#Environment file for docker-compose
+
+PATH_OMS_GLOBAL=oms-global/docker/
+PATH_OMS_CORE=oms-core/docker/
```

Comment:
Would that really be worth it? I imagine we might very soon add other environment variables to this file, I do not really see harm specifying this here.